server/sonar-server/src/test/java/org/sonar/server/measure/persistence/MeasureDaoTest.java

```diff
@@ -37,13 +37,12 @@
 public class MeasureDaoTest extends AbstractDaoTestCase {
 
   DbSession session;
-
-  MeasureDao dao;
+  MeasureDao sut;
```

Comment:
Ok, thanks for the explanation. Indeed, the concept is interesting, but it's really hard to understand the first times you read it.
But indeed it's a good idea.